amida-tech/indaba-client
# Bug/INBA-546 Custom modal button labels

#### What does this PR do?
1. Adds support to `Modal` for setting custom label text for the primary button
1. Changes particularly inaccurate confirm button labels to be more accurate (Delete, Remove, Confirm)

#### Related JIRA tickets:
[INBA-546](https://jira.amida-tech.com/browse/INBA-546)

#### How should this be manually tested?
Open confirmation modals for deleting resources and check that the primary button has an accurate label. E.g., deleting users, subjects, user groups, and deactivating a project

#### Background/Context

#### Screenshots (if appropriate):
